.travis.yml

```diff
@@ -0,0 +1,34 @@
+os: linux
+dist: bionic
+language: bash
+services: docker
+
+branches:
+  only:
+  - devel
```

Comment:
We can have this only on `master` and run only on pushing tags.

Reply:
Updated the change.

Reply:
I do not see on event tag. The images should be built and pushed only when a tag is pushed.